### PR TITLE
Updates to YAML header and added alt text to figures.

### PR DESCRIPTION
**Pull request by Fe-Sa**

### CODE REVIEW

**Comment by donal-at-NEON** [comment]
Hey @sokole , would you please review the 'fig.cap' properties for your two biodiversity tutorials? This 'fig.cap' is essentially the 'alt text' that is used by Screen Reader software to interpret graphics for people with limited visibility. Please @ @Fe-Sa and myself with any comments. Thank you!

File: tutorials-in-development/Staging_Area/02_ecocomDP_workflow_with_NEON_algae.Rmd [hunk 1]
```diff
@@ -201,7 +201,7 @@ alg_observations_with_taxa %>% head()
 
 We can also make a quick plot to see which taxon rank (i.e., what level of taxonomic specificity was achieved by the expert taxonomist) is most common:
 
-```{r plot-taxon-rank}
+```{r plot-taxon-rank, fig.cap= "Bar plot showing counts of algae taxonomic ranks at the Arikaree River site."}
```
Comment (sokole) [inline tutorials-in-development/Staging_Area/02_ecocomDP_workflow_with_NEON_algae.Rmd:204]: "Bar plot showing the frequency of each taxonomic rank observed in algae count data from the Arikaree River site."

**Comment by sokole** [comment]
@donal-at-NEON captions look good to me. I just had one suggested edit commented above.

File: tutorials-in-development/Staging_Area/02_ecocomDP_workflow_with_NEON_algae.Rmd (hunk 1 above)
Comment (Fe-Sa) [reply tutorials-in-development/Staging_Area/02_ecocomDP_workflow_with_NEON_algae.Rmd]: Thanks for reviewing, @sokole . Updated.

**Comment by donal-at-NEON** [comment]
Excellent, thank you @Fe-Sa for the update, and @sokole for the review!

**Pull request merged**